openhands-sdk/openhands/sdk/logger/logger.py

```diff
@@ -83,7 +83,7 @@ def disable_logger(name: str, level: int = logging.CRITICAL) -> None:
 
 # Quiet chatty third-party loggers
 for name in ["litellm", "LiteLLM", "openai"]:
-    disable_logger(name, logging.DEBUG if _ENABLE_LITELLM_DEBUG else logging.WARNING)
+    disable_logger(name, logging.DEBUG if _ENABLE_LITELLM_DEBUG else logging.ERROR)
```

Comment:
I'm a little bit confused, wouldn't INFO logs already be suppressed if the logging level is WARNING?